packages/connectors/src/MediaConnector.ts

```diff
@@ -43,4 +67,5 @@ export type MediaConnectorCapabilities = {
     detail: boolean;
     filtering: boolean;
     metadata?: boolean;
+    analyzeSmartCrop?: boolean;
```

Comment:
if we assume that `analyze` is going to be executed not only for smartCrop, I would consider it as object instead, like
```
analyze?: {
 smartCrop: boolean
}
```
```

Reply:
I feel like nesting it is not required

Reply:
but now it's not in sync with the capabilities...

Reply:
Sure it is 🤔